garage-door-GDOv2-Q.yaml

```diff
@@ -89,7 +89,7 @@ substitutions:
   status_led: GPIO18
 
 external_components:
-  - source: github://konnected-io/konnected-esphome@master
+  - source: github://konnected-io/konnected-esphome@door-op-fixes
```

Comment:
need to remove the branch changes before commit

Reply:
for sure